docs/general/manage-tokens/faq.mdx

```diff
@@ -462,6 +462,23 @@ exchange support and provide them the link to your account on the
 [Oasis Scan](https://www.oasisscan.com) where they can verify all transactions.
 To learn more about this issue, read the [Manage tokens](README.mdx) chapter.
 
+### Error when accessing Ledger on Linux
+
+The following error is common on Linux when accessing your Ledger device from
+the Oasis CLI or Ledger Live:
+
+```
+Error: ledger: failed to connect to device: hidapi: failed to open device
+```
+
+If you are running the application as normal user, you need to add some udev
+rules to fix the USB device permissions. The Ledger team prepared a script
+that does it for you:
+
+```shell
+wget -q -O - https://raw.githubusercontent.com/LedgerHQ/udev-rules/master/add_udev_rules.sh | sudo bash
```

Comment:
Running things from the internet via `sudo` seems quite dangerous. The script is minimal, maybe just reproduce it here or use a stable reference or something? There is also the `ledger-wallets-udev` package on Ubuntu which maybe installs the same rules?